tests/phpunit/tests/Checker/Checks/File_Type_Check_Tests.php

```diff
@@ -98,4 +98,33 @@ public function test_run_without_any_file_type_errors() {
 		$this->assertEmpty( $errors );
 		$this->assertSame( 0, $check_result->get_error_count() );
 	}
+
+	public function test_run_with_badly_named_errors() {
+		// Test plugin without any forbidden file types.
+		$check_context = new Check_Context( UNIT_TESTS_PLUGIN_DIR . 'test-plugin-file-type-badly-named-files-errors/load.php' );
+		$check_result  = new Check_Result( $check_context );
+
+		$check = new File_Type_Check();
```

Comment:
See above: Here you would then pass `File_Type_Check::TYPE_BADLY_NAMED`, so that the test only tests _this_ specific part.

Reply:
nice!